.taskfiles/Taskfile_chart.yml

```diff
@@ -0,0 +1,50 @@
+---
+version: '3'
+
+vars:
+  GIT_ROOT:
+    sh: git rev-parse --show-toplevel
+
+tasks:
+
+  # TODO: Use true gotemplate tools
+  create:
+    desc: create new chart
+    cmds:
+    - cp -r {{.GIT_ROOT}}/charts/.template {{.GIT_ROOT}}/charts/{{.CHART}}
+    - go-replace -s '${CHARTNAME}' -r "{{.CHART}}" --path={{.GIT_ROOT}}/charts/{{.CHART}} --path-pattern='*.*'
+    - task: dependency
+    status:
+    - test -d {{.GIT_ROOT}}/charts/{{.CHART}}
+    deps:
+    - check-chart
+
+
+  dependency:
+    cmds:
+    - test -d {{.GIT_ROOT}}/charts/{{.CHART}}/Chart.lock && rm {{.GIT_ROOT}}/charts/{{.CHART}}/Chart.lock || exit 0
+    - test -d {{.GIT_ROOT}}/charts/{{.CHART}}/tmpcharts && rm -rf {{.GIT_ROOT}}/charts/{{.CHART}}/tmpcharts || exit 0
+    - cd {{.GIT_ROOT}}/charts/{{.CHART}} && helm dependency update
+    silent: true
+
+  lint:
+    desc: lint your chart code
+    cmds:
+    - cd {{.GIT_ROOT}}/charts/{{.CHART}} && helm lint
+    deps:
+    - dependency
+    - check-chart
+
+  test:
+    desc: test your chart code
+    cmds:
+    - docker run --rm -it --user $(id -u):$(id -g) -e "HELM_CONFIG_HOME=/tmp/helm" -e "HELM_CACHE_HOME=/tmp/helm" -v {{.GIT_ROOT}}:/ci -w /ci quay.io/helmpack/chart-testing:latest ct lint --charts charts/{{.CHART}} --config /ci/.github/ct.yaml
```

Comment:
Should we put something in place that makes sure docker is installed/running? Not sure if Taskfile can handle that though

Reply:
It's already implemented :) => https://github.com/k8s-at-home/charts/blob/e5ced6741d322b92287db7491e1f5451b3facc70/.taskfiles/Taskfile_linux.yml#L23

Reply:
Whoops 😄